pkgs/development/python-modules/ducc0/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchgit, pytest, pybind11, numpy }:
+
+buildPythonPackage rec {
+  pname = "ducc0";
+  version = "0.23.0";
+  myversion = "0_23_0";
+
+  src = fetchgit {
```

Comment:
There is [`fetchFromGitLab`](https://nixos.org/manual/nixpkgs/stable/#fetchfromgitlab) could be used here to get the source.

Reply:
done.